libs/knowledge-store/ragstack_knowledge_store/graph_store.py

```diff
@@ -353,7 +357,7 @@ def add_nodes(
                 link_to_tags = set()  # link to these tags
                 link_from_tags = set()  # link from these tags
 
-                for tag in get_link_tags(metadata):
+                for tag in _links:
```

Comment:
Why underscore? I usually read that as unused? If this is to avoid shadowing, I think we could call things `link_sets` and `link_set` in this method.

Reply:
Underscore-something is not the convention for unused, rather for private args.
But I agree we can rename better here.